{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/password.jinja

```diff
@@ -1,25 +1,25 @@
 {% from 'forms/attrs.jinja' import form_attrs %}
 
 {% macro password(name='password', value=None, model="password", attrs={}) %}
```

Comment:
would be nice to have a "repeat" flag to handle the common case of having to enter a password twice